cargo-miri/Cargo.lock

```diff
@@ -212,9 +212,9 @@ dependencies = [
 
 [[package]]
 name = "rustc-build-sysroot"
-version = "0.4.2"
+version = "0.4.3"
```

Comment:
I released 0.4.4 just a few minutes later since clippy pointed out a potential issue. Can you update to that?